components/lib/crash/src/main/java/mozilla/components/lib/crash/notification/CrashNotification.kt

```diff
@@ -89,13 +89,14 @@ internal class CrashNotification(
             sdkLevel: Int = Build.VERSION.SDK_INT
         ): Boolean {
             return when {
-                // We always want to show the notification for background child process crashes.
-                crash is Crash.NativeCodeCrash && crash.processType ==
-                    Crash.NativeCodeCrash.PROCESS_TYPE_BACKGROUND_CHILD -> true
-
                 // We can always launch an activity from a background service pre Android Q.
                 sdkLevel < NOTIFICATION_SDK_LEVEL -> false
 
+                // We may not be able to launch an activity if a background process crash occurs
```

Comment:
nit: this is just a comment change?  Looks like it was moved.

Reply:
It's deliberately been moved, as suggested by Christian. This is so that we prefer to show the prompt when we are able to do so reliably (on old SDK versions).